src/pages/module/ssl/edit/index.tsx

```diff
@@ -106,4 +124,4 @@ const Detail: React.FC = () => {
   );
 };
 
-export default Detail;
+export default Edit;
```

Comment:
Please keep using `Detail` as the template for `Create` & `Edit`.

Reply:
Ok, I will change them.